### PR TITLE
added ownership to /opt/pega/prweb

### PR DESCRIPTION
**Pull request by shashikant-koder**
Not able to access /opt/pega/prweb in the container.

https://github.com/pegasystems/docker-pega-web-ready/issues/79

### CODE REVIEW

**Review by Saurabh-16 — APPROVED** [review]

**Comment by shashikant-koder** [comment]
@dcasavant Can you please review this.

**Review by dcasavant — COMMENTED** [review]
Question posted inline

File: tests/pega-web-ready-testcases.yaml [hunk 1]
```diff
@@ -48,51 +50,71 @@ fileExistenceTests:
...
 - name: 'Secrets File Path'
   path: '/opt/pega/secrets'
   shouldExist: true
   permissions: 'drwxrwxr-x'
+  uid: 9001
+  gid: 0
 
 - name: 'StreamVolume File Path'
-  path: '/opt/pega/secrets'
+  path: '/opt/pega/streamvol'
```
[the remark this answers is not in the document]
Comment (shashikant-koder) [reply tests/pega-web-ready-testcases.yaml:106]: I am not sure about as of but will check with team on this.

[the remark this answers is not in the document]
Comment (shashikant-koder) [reply tests/pega-web-ready-testcases.yaml:106]: @dcasavant  When pega-stream is mounting to /opt/pega, all the pre-existing folders were being cleared in that location. So, we added a separate folder streamvol to mount to that location. Fix made in BUG-481713

[the remark this answers is not in the document]
Comment (dcasavant) [reply tests/pega-web-ready-testcases.yaml:106]: It makes sense to have a specific directory for stream *(not just /opt/pega). Why do we need `/opt/pega/streamvol` *and* `/opt/pega/kafkadata`? It seem like [the readme](https://github.com/pegasystems/docker-pega-web-ready#mount-points) may be out of sync with this too.

[the remark this answers is not in the document]
Comment (shashikant-koder) [reply tests/pega-web-ready-testcases.yaml:106]: To unblock this PR, we will create an issue for this and do proper investigation on /opt/pega/streamvol and /opt/pega/kafkadata.

**Review by MadhuriArugula — APPROVED** [review]

**Pull request merged**